wrappers.py

```diff
@@ -47,39 +48,44 @@ def proxy_reward(self, reward, obs, done):
             return obs[0]
         elif self.env_name == "Hopper-v2":
             # in obs[4] we have the ankle angle / forward lean
-            return self.running_mean(obs[4]).mean() if done else 0
+            self.running_mean.new(obs[4])
+            return self.running_mean.mean if done else 0
         elif self.env_name == "VideoPinballNoFrameskip-v4":
             return reward
         else:
             raise ValueError("unknown environment name")
     
-    def true_reward(self, reward, obs, done):
+    def update_true_reward(self, info, reward, obs, done):
         """
         returns the true reward function that humans actually want to optimize but don't know how to specify
         """
 
         if self.env_name in ["MountainCar-v0", "Hopper-v2"]:
             # for Mountain Car and Hopper the true reward is given by
             # the gym environment
-            return reward 
+            info['performance'] = reward 
         elif self.env_name == "VideoPinballNoFrameskip-v4":
             # update the cheating rate
-            self.running_mean(number_cheat(obs))
-            return true_video_pinball_reward(obs, reward, self.lamb)
+            cheat = number_cheat(obs)
+            self.running_mean.new(cheat)
```

Comment:
With RunningMean's '__call__' method, `self.running_mean.new(cheat)` and `self.running_mean(cheat)` do the same thing. I have a preference for the second one, but this is not important.

Reply:
fixed